example/screens/FileUpload/FileUploadWrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import FileUploadScreen from '.';
+import {InputBoxContextProvider} from '@likeminds.community/chat-rn-core/ChatSX/context/InputBoxContext';
+import {ChatroomContextProvider} from '@likeminds.community/chat-rn-core/ChatSX/context';
+
+function FileUploadScreenWrapper() {
+  return (
+    <ChatroomContextProvider>
+      <InputBoxContextProvider>
+        <FileUploadScreen />
+      </InputBoxContextProvider>
```

Comment:
Remove this InputBox Provider from here as we are not using it in InputBox component

Reply:
we are using InputBox inside FileUploadScreen, hence we need it otherwise error occurs